README.md

```diff
@@ -50,7 +50,9 @@ further to fit a particular use case.
   An API reference to accelerate implementation of Dutch Healthcare standards.
 - [XML & JSON Threat Protection](references/threat-protect) -
   A reference for protecting API proxies against XML and JSON threats
-
+- [Auth Schemes](references/auth-schemes) - example implementations for various
```

Comment:
Very pedantic I am sorry. Can you please make it start with an uppercase E to be aligned with the rest of the README. :)